packages/legend-application-studio/src/stores/EditorStore.ts

```diff
@@ -1262,41 +1261,41 @@ export class EditorStore implements CommandRegistrar {
     );
   }
 
-  // TODO: to be removed when we process editor states properly
-  reprocessElementEditorState = (
-    editorState: EditorState,
-  ): EditorState | undefined => {
-    if (editorState instanceof ElementEditorState) {
-      const correspondingElement =
-        this.graphManagerState.graph.getNullableElement(
-          editorState.element.path,
+  /**
+   * This method helps to open the editor states which were open we updated the application with the
+   * new graph
+   */
+  buildOpenedEditorStates = (
+    openedElementEditorPaths: string[],
+    otherOpenedEditorStates: FileGenerationViewerState[],
```

Comment:
Seems like `otherOpenedEditorStates` only refers to opened generated files so maybe `openedGeneratedFiles`